src/components/MultiSelectOption.js

```diff
@@ -29,6 +29,7 @@ export default class MultiSelectOption extends Component {
           rrs__option
```

Comment:
This `<li>` needs an aria prop now I reckon `aria-disabled={option.disabled ? 'true' : 'false'}`